release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -312,6 +312,12 @@ With this release, some Operators managed by Operator Lifecycle Manager (OLM) on
 For more information, see _Token authentication for Operators on cloud providers_.
 //xref:../operators/operator_sdk/osdk-token-auth.adoc#osdk-token-auth[Token authentication for Operators on cloud providers].
 
+==== Authentication Operator honors `noProxy` when configured
+
+With this release, when `noProxy` is set, the Authentication Operator will perform connection checks directly through the configured ingress route, bypassing the cluster-wide proxy. Previously, the connection was always attempted through the cluster-wide proxy first, regardless of this setting. 
```

Comment:
Per the [implementation actuality](https://github.com/openshift/cluster-authentication-operator/pull/608/files#diff-067c8061203aef85d6b9fcc6819fda8a9cc8a490a46f38973e0d7e40e0b1c5c7R103-R104), your statement is only true when the route is reachable without the proxy. Even if noProxy is set, if the route has to be through the proxy to be reachable, the Operator will still perform the checks. So, suggesting below:
```suggestion
With this release, when `noProxy` is set and if the route is reachable without the cluster-wide proxy, the Authentication Operator will perform connection checks directly through the configured ingress route, bypassing the cluster-wide proxy. Previously, the connection was always attempted through the cluster-wide proxy first, regardless of this setting. 
```

Reply:
Thank you!